src/lib/plugin.js

```diff
@@ -19,7 +19,7 @@ const find = require('lodash/find')
 const backoffMin = 1000
 const backoffMax = 30000
 
-const REQUIRED_LEDGER_URLS = [ 'transfer', 'transfer_fulfillment', 'transfer_rejection', 'account', 'account_transfers', 'message' ]
+const REQUIRED_LEDGER_URLS = [ 'transfer', 'transfer_fulfillment', 'transfer_rejection', 'account', 'websocket', 'message' ]
```

Comment:
Oh I misunderstood, I thought it was to be used by the plugin if it were going to be included in a browser app